org.mixedrealitytoolkit.input/Interactors/Ray/MRTKRayInteractor.cs

```diff
@@ -79,14 +142,56 @@ public class MRTKRayInteractor :
 
         #region IHandedInteractor
 
-        Handedness IHandedInteractor.Handedness => (xrController is ArticulatedHandController handController) ? handController.HandNode.ToHandedness() : Handedness.None;
+        Handedness IHandedInteractor.Handedness
+        {
+            get
+            {
+                #pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
Limit the scope of the warning disable...it should only cover controller usage, and nothing else.  PLease follow the pattern I used in my draft PR 

https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/745/files